osquery/tables/system/windows/chassis_info.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+QueryData genChassisInfo(QueryContext& context) {
+  QueryData results;
+
+  WmiRequest wmiSystemReq("select * from Win32_SystemEnclosure");
+  const auto& wmiResults = wmiSystemReq.results();
+
```

Comment:
I notice this removed `wmiSystemReq.getStatus(). ok() ` Is that okay? I guess we're checking if `wmiResults` is empty... Is there ever a time `wmiResults` is not empty, but `wmiSystemReq.getStatus` is an error?

Reply:
I will double check, but any time getStatus errors, wmiResults should be empty.